packages/material-ui-lab/src/TreeItem/TreeItem.js

```diff
@@ -16,38 +16,48 @@ export const styles = theme => ({
     padding: 0,
     outline: 0,
     WebkitTapHighlightColor: 'transparent',
-    '&:focus > $content': {
-      backgroundColor: theme.palette.grey[400],
+    '&:focus > $content $label': {
```

Comment:
I don't remember the outcome of the following topic: shouldn't the states fully expand horizontally?
It behaves this way with Storybook, Gmail/Google Drive, VS code or macOS tree views. What's the motivation for the current cut approach?

![Capture d’écran 2020-02-01 à 16 43 50](https://user-images.githubusercontent.com/3165635/73594810-10b13c00-4512-11ea-972d-54b4c276ea8a.png)
![Capture d’écran 2020-02-01 à 16 42 41](https://user-images.githubusercontent.com/3165635/73594793-e52e5180-4511-11ea-9dc8-e7ab7ad28044.png)
![Capture d’écran 2020-02-01 à 16 41 18](https://user-images.githubusercontent.com/3165635/73594776-b617e000-4511-11ea-8553-c15f58184a87.png)
![Capture d’écran 2020-02-01 à 16 41 57](https://user-images.githubusercontent.com/3165635/73594828-3f2f1700-4512-11ea-8dbe-7f3a4f63834e.png)

Reply:
This is challenging to do with list items since the browser uses padding to indent them.

I think the best way I've seen it done is using a pseudo element but essentially it's just a hack.
I guess it would be nice to do but I'm unsure if there's a good way to do it?

Reply:
I see, the approach used in https://material-ui.com/components/tree-view/#gmail-clone doesn't work with more nesting (by the way, it doesn't seem to work any longer in this pull request).

The only alternative I can think of is to add a "padding increment" prop to the component and to have the right padding automatically applied based on the nesting level. Probably not something to consider in this pull request! It sounds off-topic, sorry O:).